src/features/content-answers/ContentAnswers.tsx

```diff
@@ -15,10 +21,10 @@ const ContentAnswers = () => {
 
     return (
         <div>
-            <ContentAnswersOpenButton onClick={handleClick} />
+            <ContentAnswersOpenButton onClick={handleClick} fileExtension={fileExtension} />
         </div>
     );
 };
 
-export type ContentAnswersProps = ExternalProps;
+export type ContentAnswersProps = ExternalProps & Props;
```

Comment:
is this change still needed for how we're currently using `ContentAnswersProps`?